### DISCUSSION TITLE
Possible to map seasons from one show to another show?

### DISCUSSION THREAD

**Discussion by lightmaster** [meisnate12/Plex-Meta-Manager#2054] [Q&A]
I just discovered Kometa so still trying to get a feel for what it's capable of doing. Is it possible to pull metadata from one show into another show? For instance, I'm trying to fix how Doctor Who (2005) Season 14 is showing up as Season 1 of a new show instead of continuation of the previous show. Is it possible to essentially map Doctor Who (2024) Season 1 to Doctor Who (2005) Season 14, then 2024 Season 2 to 2025 Season 15, etc? Or is the only option to manually create the metadata file and hard code the metadata from Season 14 to have the titles and summaries it needs?

**Comment by chazlarson** [comment]
Doing this with a metadata file is the only solution here AFAIK.  There's no way for Kometa to tell Plex, "You should match this thing to this other non-existent thing".  

You'll probably have to store it on disk as Season 14 and 15 in the same folder as the old show and then use a metadata file to set the metadata.
